drivers/backup/backup.go

```diff
@@ -280,7 +280,7 @@ type SchedulePolicy interface {
 	DeleteBackupSchedulePolicy(orgID string, policyList []string) error
 
 	// GetSchedulePolicyUid gets the uid for the given schedule policy
-	GetSchedulePolicyUid(orgID string, tx context.Context, schPolicyName string) (string, error)
+	GetSchedulePolicyUid(orgID string, ctx context.Context, schPolicyName string) (string, error)
```

Comment:
Change the variable name to `schedulePolicyName` to make it clear

Reply:
Done.